lesson01/assignment/script-2.js

```diff
@@ -0,0 +1,108 @@
+// Search the "form", and handle its submit event
+const form = document.querySelector('#first-form');
+form.addEventListener('submit', function (event) {
+    event.preventDefault()
+    // create an array to transform or pass the user data, here an empty array
+    let data = [ 'love' , 'peace', '&', 'harmony']
+   
+    //Target the input field to get data from the user
+    const dataUser = document.querySelector('form .entry');
+
+    // Get the user's data
+    const grabDataUser = dataUser.value;
+
+    // Print the output in the form plus a text, and display the data string
+    document.querySelector('form span').innerHTML = `${'This is what you get '}` + ' " ' + grabDataUser + ' " ' + data.toString(); 
+
+    // Slice method
+    // Only works with words that contains at least 5 characters
+    // Targest "slc1" button, and handle its click event
+    slc1.addEventListener('click', function (e) {
+        e.preventDefault();
+
+        // slice method applied from the user's data
+        let slc = grabDataUser.slice(0, -4);
+        
+        // Print the output
+        document.getElementById('slc').innerHTML =  slc;  
+    })
+})
+
+
+// Join method
+// Search the "form2", and handle its submit event
+const form2 = document.querySelector('#second-form');
+form2.addEventListener('submit', function (evento) {
+    evento.preventDefault()

```

Comment:
Because the event is the default argument to a form submit function, it can be called anything you like, as long as it's the same both places, as you've found here. But english speaking developers traditionally use either 'e' or 'event'. Just a thing to keep in mind as you move into a professional setting, depending on where you work 😄